devicetypes/smartthings/testing/simulated-window-shade.src/simulated-window-shade.groovy

```diff
@@ -0,0 +1,250 @@
+/**
+ *  Copyright 2018, 2019 SmartThings
+ *
+ *  Provides a simulated window shade.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+metadata {
+	definition (name: "Simulated Window Shade", namespace: "smartthings/testing", author: "SmartThings", runLocally: false, mnmn: "SmartThings", vid: "generic-window-shade") {
+		capability "Actuator"
+		capability "Window Shade"
+		capability "Window Shade Preset"
+		//capability "Switch Level"
+
+		// Commands to use in the simulator
+		command "openPartially"
+		command "closePartially"
+		command "partiallyOpen"
+		command "opening"
+		command "closing"
+		command "opened"
+		command "closed"
+		command "unknown"
+	}
+
+	preferences {
+		section {
+			input("actionDelay", "number",
+				title: "Action Delay\n\nAn emulation for how long it takes the window shade to perform the requested action in seconds (1-120; default if empty: 5 sec).",
```

Comment:
I think you can remove the part at the end since it's already in the `description`

Reply:
Done!